spec/unit/user_config_spec.rb

```diff
@@ -12,11 +12,30 @@ def set_file_path(path)
       config.reload
     end
 
-    before(:each) do
-      set_file_path("missing")
+    def with_env(vars)
+      original = ENV.to_hash
+      vars.each { |k, v| ENV[k] = v }
+
+      begin
+        yield
+      ensure
+        ENV.replace(original)
+      end
+    end
+
+    it "preferably uses SKYLIGHT_USER_CONFIG_PATH" do
+      dir = File.expand_path(Dir.mktmpdir)
+      config_path = File.join(dir, 'skylight_user_config_path')
+      FileUtils.touch(config_path)
```

Comment:
I didn't realize at first that the config path was actually a path to the config file, not a path to a folder used for config :(.  I could make that more clear in the example config_path variable name and value.